### ISSUE TITLE
A11y_AzureDataStudio_CreateAzureKubernetesServiceCluster_ScreenReader:NVDA does not announces the title of the window.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/azuredatastudio#7622]
"[Check out Accessibility Insights!](https://accessibilityinsights.io) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**#A11y_ADS_OctTestPass;#A11yMAS; #A11yTCS;#SQLAzureDataStudio;#MAS1.3.1;#DesktopApp;#NVDA;#JAWS;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
NVDA Version:2019.2

**Repro Steps:**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on 
        select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button and again click on next 
        button and click on script to Notebook.
8.	Create Azure Kubernetes service cluster window screen is opened.
9. Check the NVDA screen reader  does not announces the title of the window or not.

**Actual:**
NVDA does not announces the title of the window as soon as window receives screen reader focus.

**Expected:**
NVDA should announce the title of the window as soon as window receives screen reader focus.

**Same issue observed in JAWS.**

**Same issue observed for the below scenario:**
**1.Prerequisites**
    1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button and again click on next button and click on script to Notebook ->Navigate till Prerequisites.
8.	Verify that all the elements of " Prerequisites" are accessible and MAS Compliant.

**2.Check dependencies** 
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button and again click on next button and click on script to Notebook ->Navigate till Check dependencies.
8.	Verify that all the elements of " Check dependencies " are accessible and MAS Compliant

**User Impact:**
Screen reader user will not understand the name of the regarding the control if it does not announces the related name about the control.

**Attachment for Reference:**
[7622_A11y_AzureDataStudio_CreateAzureKubernetesServiceCluster_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3734619/7622_A11y_AzureDataStudio_CreateAzureKubernetesServiceCluster_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
another notebook related issue. @corivera @chlafreniere

**Comment by Charles-Gagnon** [comment]
@vinodbk197 This seems to be fixed with my testing - NVDA is announcing the title correctly when the Notebook is opened.

```
Script to Notebook
button
Azure Data Studio Dev  document
 
deploy-bdc-aks - Azure Data Studio Dev
The editor is not accessible at this time. Press Alt+F1 for options.  edit  has auto complete  multi line
```

I'm going to close this for now - let me know if you're still seeing this issue on your end.

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;
